config/playground/jobs/sample_data_tooling/startup_deployment.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: initial-startup-job
```

Comment:
this name should be more detail functions.

Reply:
`initial-startup-job` changed to `sample-data-startup-job`